doc/classes/NavigationRegion2D.xml

```diff
@@ -43,10 +43,18 @@
 				Returns the current navigation map [RID] used by this region.
 			</description>
 		</method>
-		<method name="get_region_rid" qualifiers="const">
+		<method name="get_region_rid" qualifiers="const" is_deprecated="true">
 			<return type="RID" />
 			<description>
 				Returns the [RID] of this region on the [NavigationServer2D]. Combined with [method NavigationServer2D.map_get_closest_point_owner] can be used to identify the [NavigationRegion2D] closest to a point on the merged navigation map.
+				[i]Deprecated.[/i] Use [method get_rid] instead.
+			</description>
+		</method>
+		<method name="get_rid" qualifiers="const">
+			<return type="RID" />
+			<description>
+				Returns the [RID] of this region on the [NavigationServer2D].
+				Combined with [method NavigationServer2D.map_get_closest_point_owner] can be used to identify the [NavigationRegion2D] closest to a point on the merged navigation map.
```

Comment:
Don't think we need to add a new paragraph here. Also the description of the old method can probably be reduced to just the first sentence.